app/views/courses/_cohort_courses.html.erb

```diff
@@ -17,7 +17,8 @@
     </div>
     <div class="cohort-select">
       <select class="cohorts">
-        <option selected disabled><%= t("cohort.view_other") %></option>
+        <option selected disabled>
+        <%= Features.wiki_ed? ? t("courses.view_other") : t("courses_generic.view_other") %></option>
```

Comment:
Actually, this is the only erb file where this pattern shows up. The other instances are html.haml and jsx files. If the method is added in /app/helpers/course_helper.rb (where I put it), will it be accessible to JSX and HTML files too?